compliance/specs/exceptions/rfc5746/4.4.toml

```diff
@@ -7,8 +7,7 @@ servers nevertheless do permit it, they MUST follow the requirements
 in this section.
 '''
 reason = '''
-s2n-tls servers do not support renegotiation
-and do not allow clients to renegotiate.
+s2n-tls servers do not support renegotiation.
```

Comment:
This was correct as-is. s2n-tls clients can support renegotiation, but servers do not support renegotiation. Same for the others.

Reply:
Oh I misread the original as "s2n-tls clients do not allow renegotiation". I'll revert this change. Thank you for pointing out